### PR TITLE
Remove occurences of gitlab

### PR DESCRIPTION
**Pull request by CherryKitten**
Includes a version bump of the openldap container because of the switch to nexus

### CODE REVIEW

File: roles/postgresql_client_access/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # `famedly.base.postgresql_client_access` ansible role for configuring users, databases and pg_hba entries of an PostgreSQL instance
 This convenience role creates and updates users, databases and pg_hba entries.
-It's designed to work with PostgreSQL servers running inside a docker container deployed by [famedly.base.postgresql](https://gitlab.com/famedly/infra/collections/base/-/tree/main/roles/postgresql).
+It's desiged to work with PostgreSQL servers running inside a docker container deployed by [famedly.base.postgresql](https://github.com/famedly/ansible-collection-base/tree/main/roles/postgresql).
```
Comment (jcgruenhage) [inline roles/postgresql_client_access/README.md:3]: I think you accidentally deleted a character here?

File: roles/postgresql_client_access/README.md [hunk 2]
```diff
@@ -86,4 +86,4 @@ Docker needs to be installed and configured.
 GNU Affero General Public License v3
 
 ## Author Information
-Famedly GmbH, famedly.de
\ No newline at end of file
+Famedly GmbH, famedly.de
```
Comment (jcgruenhage) [inline roles/postgresql_client_access/README.md:89]: I'm a bit blind, where's  the diff here?

**Review by jcgruenhage — CHANGES_REQUESTED** [review]

File: roles/postgresql_client_access/README.md (hunk 1 above)
Comment (CherryKitten) [reply roles/postgresql_client_access/README.md:3]: Nope, github doesn't have the minus on those urls that gitlab does, so this is correct

File: roles/postgresql_client_access/README.md (hunk 2 above)
Comment (CherryKitten) [reply roles/postgresql_client_access/README.md:89]: Doing a local diff it shows that there was no newline at the end of the file before, no idea why that doesn't show in the github diff

Comment (jcgruenhage) [reply roles/postgresql_client_access/README.md:89]: GitLab nicely showed that with a little indicator, and I thought github does the same, but apparently not.

File: roles/postgresql_client_access/README.md (hunk 1 above)
Comment (jcgruenhage) [reply roles/postgresql_client_access/README.md:3]: I meant the "designed" vs "desiged" in the beginning there, not the URLs

Comment (CherryKitten) [reply roles/postgresql_client_access/README.md:3]: oh. whoops, sorry about that

Comment (CherryKitten) [reply roles/postgresql_client_access/README.md]: fixed

Comment (jcgruenhage) [reply roles/postgresql_client_access/README.md]: :+1:

**Review by jcgruenhage — APPROVED** [review]

**Pull request merged**